addon/ng2/utilities/get-dependent-files.ts

```diff
@@ -0,0 +1,105 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as ts from 'typescript';
+
+interface MapType {
```

Comment:
The name is misleading. Maybe using something like `ImportMap` (no need for the type suffix)?

Your keys can be the path of the file, the value an array of relative/absolute paths of imports. What do you think?